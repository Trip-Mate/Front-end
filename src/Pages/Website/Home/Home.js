import React, { Fragment } from 'react'

function Home() {
  return (
    <Fragment>home</Fragment>
  )
}

export default Home
