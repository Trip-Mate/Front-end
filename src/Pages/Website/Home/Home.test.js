import { shallow } from "enzyme";
import toJSON from "enzyme-to-json";
import React from "react";

import Home from "./Home";

test("Home snapshot matches", () => {
  const wrapper = shallow(<Home />);

  expect(toJSON(wrapper)).toMatchSnapshot();
});