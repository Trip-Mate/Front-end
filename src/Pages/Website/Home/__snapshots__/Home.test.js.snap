// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home snapshot matches 1`] = `
<Fragment>
  home
</Fragment>
`;
